charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -17,14 +17,6 @@ rules:
 {{- end }}
     verbs:
       - create
-  # The CNI plugin needs to get pods, nodes, and namespaces.
```

Comment:
I think calico/node also needs some of these, unless they are already covered somewhere else.